upf/upf_pfcp.c

```diff
@@ -2630,6 +2630,20 @@ format_pfcp_session (u8 * s, va_list * args)
 	      vlib_time_now (gtm->vlib_main) - sx->last_ul_traffic,
 	      rules->inactivity_timer.handle);
 
+  if (gtm->pfcp_spec_version == 16)
+    {
+      s = format (s, "TEID assignment per choose ID\n");
+      u16 idx = 1;
+      do
+	{
+	  u32 teid = *sparse_vec_elt_at_index (sx->teid_by_chid, idx);
+	  if (teid)
+	    s = format (s, "Choose ID [%u] TEID [%u]\n", idx, teid);
```

Comment:
1.) indent that properly, 2 spaces more than the header in line 2635
2.) its a bit verbose maybe:
```
s = format (s, "    %3u: %u (0x%08x)\n", idx, teid, teid);
```

Reply:
Sorry, completely forgot about output indentation
fixed